### PR TITLE
Fixing Typo - Unnecessary repetition

### PR DESCRIPTION
**Pull request by creationlayer**
Improving bottom description, and fixing facebook link, sorry for multiple commits in request.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -881,7 +881,7 @@ <h2 class="section-title slogan" style="opacity: 1;"><span>The Future of Digital
                     <h2 class="parallax-title"><span>Contact us</span></h2>
 			<p>
                info<span style="display:none"></span>@viacoin.org
-                    <p class="parallax-slogan">Viacoin is an open-source Bitcoin blockchain based open-source project that integrates Clearing House. ClearingHouse is a protocol that allows for asset issuance and p2p trading on the Viacoin blockchain. Get involved on Github!</p>
+                    <p class="parallax-slogan">Viacoin is a Bitcoin blockchain based open-source project that integrates Clearing House. ClearingHouse is a protocol that allows for asset issuance and p2p trading on the Viacoin blockchain. Get involved on Github!</p>
```
Comment (btcdrak) [inline index.html]: I wonder if it's confusing to mention Bitcoin or not see brackets below.

or "Viacoin is a blockchain (based on Bitcoin) project that integrates ..."

Also moving open-source seems to exclude Viacoin from the opensource yet it is also open source.

`ClearingHouse` not two words.

Comment (creationlayer) [inline index.html]: Added the more standard description simplified from book commit.

**Comment by creationlayer** [comment]
improved the description...

**Comment by btcdrak** [comment]
Needs rebase.

**Comment by btcdrak** [comment]
merged in 1de83da

**Pull request closed without merging**